commands/commands.go

```diff
@@ -22,7 +22,7 @@ var Commands = Cmd{
 	"build":        BuildCommand(),
 	"build-server": BuildServerCommand(),
 	"bump":         common.BumpCommand(),
-	"call":         CallCommand(),
+//	"call":         CallCommand(),
```

Comment:
can we remove the command code as well here please?

Reply:
duh, more haste less speed.